packages/react/src/hooks/asset/useCreateAsset.test.ts

```diff
@@ -31,24 +31,23 @@ describe('useCreateAsset', () => {
   describe('create', () => {
     it('mutates', async () => {
       const utils = renderHook(() => useCreateAsset());
-
       const { result, waitFor } = utils;
-
       await waitFor(() => expect(result.current.mutate).toBeDefined());
-
       await act(async () => {
-        result.current.mutateAsync?.({
-          name: assetName,
-          ...getSampleVideo(),
-        });
+        result.current.mutateAsync?.([
```

Comment:
I don't think this should be required to be an array - we should be able to handle array and non-array params